types/pdfkit/index.d.ts

```diff
@@ -226,6 +465,8 @@ declare namespace PDFKit.Mixins {
         textIndent?: number | undefined;
         destination?: string | undefined;
         goTo?: string | undefined;
+        /** The structure element to add each paragraph to */
+        structParent?: PDFStructureElement | undefined;
```

Comment:
Here is where I've add structParent

Reply:
We should also add the structType option as described in https://pdfkit.org/docs/accessibility.html#automatic_marking_and_structure_construction_for_text (ReferenceSource: https://github.com/foliojs/pdfkit/blob/4ec77ddc8c090c8d0d57fbd72cff433e9ce0d733/lib/mixins/text.js#L49C60-L49C60)